sdk/storage/Azure.Storage.sln

```diff
@@ -111,6 +111,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Batch", "Batch", "{610DBA57
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Storage.Blobs.Batch.Tests", "Azure.Storage.Blobs.Batch\tests\Azure.Storage.Blobs.Batch.Tests.csproj", "{2C48AA18-94E2-4B8E-9345-DCA941B13EB1}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ApiCompat", "..\..\eng\ApiCompat\ApiCompat.csproj", "{16260507-C87E-44D2-883F-8E338B7A519F}"
```

Comment:
I assume this is to enable restore while working in VS?

Reply:
Yep

Reply:
Does the apicompat step fail if this isn't included in the solution or does it just ignore?

Reply:
No it will still run if the ApiCompat has already been restored on the local machine. If it hasn't been restored, it causes the solution build to fail.

Reply:
Fails, ResolveReferences complains about restore.